web/src/beta/components/Button/index.tsx

```diff
@@ -120,7 +126,7 @@ const StyledButton = styled.button<ButtonProps>`
     size === "medium"
       ? `${metricsSizes["s"]}px ${metricsSizes["l"]}px`
       : `${metricsSizes["xs"]}px ${metricsSizes["s"]}px`};
-  margin: ${({ margin }) => margin || `${metricsSizes["m"]}px`};
```

Comment:
Button should not have a default margin. The spacing like gap should be handled by its container.